core/i18n/src/ITwinLocalization.ts

```diff
@@ -117,10 +123,16 @@ export class ITwinLocalization implements Localization {
    * @throws Error if no keys resolve to a string.
    * @public
    */
-  public getLocalizedString(key: string | string[], options?: TOptionsBase): string {
+  public getLocalizedString(key: string | string[], options?: TranslationOptions): string {
```

Comment:
what was the type of TOptionsBase from before?

Reply:
TOptionsBase comes from i18next. It defines the different possible options that could be used by i18next's translation functions. TranslationOptions (in Localization.ts) looks to define a subset of those options that iTwin thinks should be available, but also still allows for any key-value pair to be used.

Reply:
Per our discussion, the type was reverted back to `TOptionsBase`.